go/libkb/kbsig.go

```diff
@@ -520,6 +589,10 @@ func MakeSig(
 	case KeybaseSignatureV2:
 		prevSeqno := me.GetSigChainLastKnownSeqno()
 		prevLinkID := me.GetSigChainLastKnownID()
+		hPrevInfo, err := me.GetExpectedNextHPrevInfo()
```

Comment:
you should use a different variable name than `err` here since the outer `err` is used, so this will screw up the error handling.

Reply:
just tested. can confirm. didn't realize go's order of precedence in the scope. very good to know.